otm-gateway.html

```diff
@@ -54,6 +54,24 @@ <h2>Status of This Document</h2>
     <section id="conformance">
     </section>
 
+    <section id="introduction" class="informative">
+      <h2>Introduction</h2>
+
+      <p>This specification describes APIs enabling digital object repository systems to manage preservation of content.
+      The interfaces described here are intended to fit within the larger context of the <a href="preservation-workflow.html">One to Many Preservation Workflow</a>.</p>
+      <p>This specification defines a set of interfaces for synchronous interactions for depositing, restoring,
+      and purging objects across multiple distributed digital preservation (DDP) systems, using a single system
+      managed by the repository administrators. Because DDPs (and the 
+      <a href="otm-bridge.html">One to Many Bridge API</a>) perform deposit and retrieval actions 
+      asynchronously, the Gateway's synchronous interfaces are needed to support real-time feedback for 
+      repository managers and curators. The Gateway also serves as a short-term cache of content to be
+      preserved, allowing the repository system to continue its normal object life-cycle while ensuring exact
+      bitwise preservation of the objects at the time they were selected for preservation. Gateway 
+      implementations should seek to guarantee eventual preservation of objects sent for deposit.</p>
+      <p>For background, it may be helpful to read the <a href="https://wiki.lyrasis.org/display/OTM/Project+Overview+and+Goals">One to Many Project Overview and Goals</a> and the <a href="https://wiki.lyrasis.org/display/OTM/User+Stories">One to Many User Stories</a>.
```

Comment:
```suggestion
      <p>For background, it may be helpful to read the 
      <a href="https://wiki.lyrasis.org/display/OTM/Project+Overview+and+Goals">One to Many Project Overview
      and Goals</a> and the <a href="https://wiki.lyrasis.org/display/OTM/User+Stories">One to Many User Stories</a>.
```